review-tools/pick-to-branch

```diff
@@ -69,8 +69,18 @@ m*)
     ;;
 esac
 
+REMOTE=`git branch --list $TARGET -vv | sed 's/^\* //' \
+ | awk '{ sub(/\[/, "", $3); sub(/\/.*/, "", $3);  print $3 }'`
```

Comment:
The more git-ish way to do this seems to be:

``` shell
REMOTE=$(git for-each-ref --format='%(push:remotename)' \
                  $(git symbolic-ref -q HEAD))
```

Reply:
Nice, but this does not detect the remote of `$TARGET`,
and replacing `HEAD` by  `$TARGET` does not work.

Reply:
You're right...  but `git rev-parse` to the rescue!

``` shell
REMOTE=$(git for-each-ref --format='%(push:remotename)' \
             $(git rev-parse --symbolic-full-name $TARGET))
```

Reply:
Thanks - this works; improvement done.